docs/source/blogs/Best_perf_practice_on_DSR1_in_TRT-LLM.md

```diff
@@ -0,0 +1,362 @@
+- `--dataset`: Prompt dataset used to benchmark. our official benchmark dataset has ISL = 1K, OSL = 2K
+- `--backend`: Inference backend. Here we use Pytorch backend. 
+- `--tp 8`: Tensor parallel size is 8.
+- `--ep 8`: Expert parallel size is 8.
+- `--max_batch_size`: Max batch size in each rank.
+- `--max_num_tokens`: Max num tokens in each rank.
+- `--num_requests`: Num requests used for the benchmark.
+- `--concurrency`: Total concurrency for the system.
+- `--kv_cache_free_gpu_mem_fraction`: Mem fraction used to hold kv cache tokens.
+- `--extra_llm_api_options`: Used to specify some extra config. The content of the file is as follows:
+
+    ``` yaml
+        pytorch_backend_config:
+            use_cuda_graph: true
+            cuda_graph_padding_enabled: true
+            cuda_graph_batch_sizes:
+            - 1
+            - 2
+            - 4
+            - 8
+            - 16
+            - 32
+            - 64
+            - 128
+            - 256
+            - 384
+            print_iter_log: true
+            enable_overlap_scheduler: true
+        enable_attention_dp: true
+    ```
+
+
+#### Expected Result Format
+The perf might be different from different datasets and machines
+
+``` 
+===========================================================
+= PERFORMANCE OVERVIEW
+===========================================================
+Request Throughput (req/sec):                     17.5825
+Total Output Throughput (tokens/sec):             36008.8948
+Per User Output Throughput (tokens/sec/user):     11.7967
+Per GPU Output Throughput (tokens/sec/gpu):       4501.1119
+Total Latency (ms):                               2795511.9559
+Average request latency (ms):                     174119.3317
+```
+
+## H200 NVL8
+### Prerequisites
+
+``` bash
+# Prerequisites
+apt-get update && apt-get -y install git git-lfs
+git lfs install
+
+# Replace with your actual path
+YOUR_WORK_PATH=<YOUR_WORK_PATH>
+YOUR_MODEL_PATH=<YOUR_MODEL_PATH>
+YOUR_DATA_PATH=<YOUR_DATA_PATH>
+
+# Clone the TensorRT-LLM repository
+cd $YOUR_WORK_PATH
+git clone https://github.com/NVIDIA/TensorRT-LLM.git
+cd TensorRT-LLM
+git submodule update --init --recursive
+git lfs pull
+
+# Clone the DeepSeek-R1 model
+cd $YOUR_MODEL_PATH
+GIT_LFS_SKIP_SMUDGE=1 git clone https://huggingface.co/deepseek-ai/DeepSeek-R1
+git lfs pull  # Download the full model weight will take a long time
+```
+**Note**: Replace `<*_PATH>` to your actual path. 
+
+#### Build Docker 
+Create a docker and run:
+
+``` bash
+cd TensorRT-LLM
+make -C docker jenkins_run LOCAL_USER=1 DOCKER_RUN_ARGS="-v $YOUR_MODEL_PATH:$YOUR_MODEL_PATH:ro -v $YOUR_DATA_PATH:$YOUR_DATA_PATH:ro -v $YOUR_WORK_PATH:$YOUR_WORK_PATH"
+```
+Here we set `LOCAL_USER=1` argument to set up the local user account inside the container.
```

Comment:
There is no need to repeat these duplicated instructions. These should be common instructions regardless of B200/H200.